doc/api/test.md

```diff
@@ -2944,6 +2944,11 @@ defined. The corresponding declaration ordered event is `'test:start'`.
     `undefined` if the test was run through the REPL.
   * `message` {string} The diagnostic message.
   * `nesting` {number} The nesting level of the test.
+  * `level` {string} The severity level of the diagnostic message, which determines its output color in the reporter.
+    Possible values are:
+    * `'info'`: Informational messages.
+    * `'warn'`: Warnings.
+    * `'error'`: Errors.
```

Comment:
```suggestion
  * `level` {string} The severity level of the diagnostic message.
    Possible values are:
    * `'info'`: Informational messages.
    * `'warn'`: Warnings.
    * `'error'`: Errors.
```
I'm adding this suggestion since the color is a "side-effect" and happens only in the `spec` reporter.
This information could help users while creating a new custom reporter

Reply:
I will update and commit it. Thanks.

Reply:
Hi @pmarchini, Docs is updated.

Reply:
@pmarchini Please let me know if more changes is required.